pkg/k8s/watchers/resources/resources.go

```diff
@@ -8,16 +8,11 @@ package resources
 const (
 	// K8sAPIGroupServiceV1Core is the identifier for K8s resources of type core/v1/Service.
 	K8sAPIGroupServiceV1Core = "core/v1::Service"
-	// K8sAPIGroupEndpointV1Core is the identifier for K8s resources of type core/v1/Endpoint.
-	K8sAPIGroupEndpointV1Core = "core/v1::Endpoint"
 	// K8sAPIGroupPodV1Core is the identifier for K8s resources of type core/v1/Pod.
 	K8sAPIGroupPodV1Core = "core/v1::Pods"
 	// K8sAPIGroupSecretV1Cores is the identifier for K8s resources of type core/v1/Secret.
 	K8sAPIGroupSecretV1Core = "core/v1::Secrets"
-	// K8sAPIGroupEndpointSliceV1Beta1Discovery is the identifier for K8s resources of type discovery/v1beta1/EndpointSlice.
-	K8sAPIGroupEndpointSliceV1Beta1Discovery = "discovery/v1beta1::EndpointSlice"
-	// K8sAPIGroupEndpointSliceV1Beta1Discovery is the identifier for K8s resources of type discovery/v1/EndpointSlice.
-	// todo(tom): double check the uses of these two.
+	// K8sAPIGroupEndpointSliceV1Discovery is the identifier for K8s resources of type discovery/v1/EndpointSlice.
```

Comment:
Nit. I would clarify that it actually represents all types of endpoint/endpointslices.

Reply:
Renamed it to `K8sAPIGroupEndpointSliceOrEndpoint` to make it clear.